### DISCUSSION TITLE
Inline calculator not working on Windows

### DISCUSSION THREAD

**Discussion by kyrathasoft** [zim-desktop-wiki/zim-desktop-wiki#2126] [Q&A]
I've enabled inline calculator plugin, but when I try examples such as the following, I don't get a result:

3+3=

1
2
--- +

**Comment by bttrx** [comment]
1. What Zim version do you use?
2. Did you click "Tools" -> "Evaluate Math"? (_"Note that the cursor needs to be behind the expression you want to evaluate."_)

**Reply by brucewestfall** [reply]
pushing the F5 function key runs the script.
the first one you showed is the correct format.
3+3=
and then F5

**Reply by brucewestfall** [reply]
just noticed the original question was Aug 2022.  Oops.

**Reply by bttrx** [reply]
Answering late doesn't hurt here, I think.
